### PR TITLE
Added effect Low pitch

### PR DESCRIPTION
**Pull request by ProfessorBiddle**
Just like high pitch, but low pitch.

### CODE REVIEW

**Comment by Numi43** [comment]
High pitch should be called Helium Vapour.
Not sure how low pitch should be called.

**Comment by ProfessorBiddle** [comment]
> High pitch should be called Helium Vapour.
> Not sure how low pitch should be called.

Could be Sulfur Hexaflouride

File: ChaosMod/Effects/db/Misc/MiscLowPitch.cpp [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+//Effect by ProfessorBiddle, but the code is pretty much copied and pasted
+
+static int m_targetPitch;
+
+static void OnStart()
+{
+	m_targetPitch = g_random.GetRandomInt(-900, -300);
+
+	SET_AUDIO_FLAG("AllowScriptedSpeechInSlowMo", true);
+	SET_AUDIO_FLAG("AllowAmbientSpeechInSlowMo", true);
```
Comment (pongo1231) [inline ChaosMod/Effects/db/Misc/MiscLowPitch.cpp:14]: No need to set / unset those flags as the actual game speed isn't being modified

File: ChaosMod/Effects/db/Misc/MiscLowPitch.cpp [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	SET_AUDIO_FLAG("AllowScriptedSpeechInSlowMo", false);
+	SET_AUDIO_FLAG("AllowAmbientSpeechInSlowMo", false);
+}
+
+static void OnTick()
+{
+	Hooks::SetAudioPitch(m_targetPitch);
+}
+
+static RegisterEffect registerEffect(EFFECT_LOW_PITCH, OnStart, OnStop, OnTick);
```
Comment (pongo1231) [inline ChaosMod/Effects/db/Misc/MiscLowPitch.cpp:30]: Needs to be updated to include the effect info

**Review by pongo1231 — CHANGES_REQUESTED** [review]

File: ChaosMod/Effects/db/Misc/MiscLowPitch.cpp [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+static void OnTick()
+{
+	Hooks::SetAudioPitch(m_targetPitch);
+}
+
+static RegisterEffect registerEffect(EFFECT_LOW_PITCH, OnStart, OnStop, OnTick, EffectInfo
+	{
+		// These are always required, you may have to add more designators depending on your effect
+		.Name = "Low Pitch",
+		.Id = "misc_low_pitch",
```
Comment (pongo1231) [inline ChaosMod/Effects/db/Misc/MiscLowPitch.cpp:28]: The effect id here and in `Effects.cs` are different (`misc_low_pitch` and `misc_lowpitch`)

Comment (ProfessorBiddle) [reply ChaosMod/Effects/db/Misc/MiscLowPitch.cpp]: whoops, fixed

**Pull request merged**